kudos-webapps/src/main/webapp/vue-app/kudos/components/KudosApp.vue

```diff
@@ -385,7 +388,7 @@ export default {
         parentEntityId: this.parentEntityId,
         receiverType: this.receiverType,
         receiverId: this.receiverId,
-        message: this.kudosMessage
+        message: this.sanitizedKudosMessage
```

Comment:
We should keep saving the formatted text to show it in the activity stream as the user write it. 
The fix should be in the notification template.